mix.exs

```diff
@@ -47,7 +47,7 @@ defmodule RailwayUi.MixProject do
       {:gettext, "~> 0.11"},
       {:jason, "~> 1.0"},
       {:plug_cowboy, "~> 2.0"},
-      {:railway_ipc, "0.2.0"},
+      {:railway_ipc, "~> 0.2.1"},
```

Comment:
Using the twiddlw waka instead. This way we don't have to keep bumping the versions at the same time.